modules/media/help/media.md

```diff
@@ -1,51 +1,43 @@
-# Media Module
+# Media
 
-## Overview
+The Media module serves as a repository for files associated with a particular 
+candidate-timepoint or instrument in a study.  Users can access and upload files 
+such as PDF scans, recordings, log files, or stimulus presentation media in multiple 
+formats such as .pdf, .mp4, .mp3, .txt, etc. 
 
-Media module allows users to **upload**, **search** and **edit** media files associated with a specific candidate timepoint in Loris.
-Any kind of data associated with a candidate timepoint can be uploaded through this module: PDFs, videos, recordings, scripts, log files, etc. Files can optionally be associated to a specific instrument form within a given candidate timepoint.
+## Searching for a Media File
 
+Under the Browse tab, use the Selection Filters to search for files by fields such 
+as including filename, file type, candidate PSCID, Visit Label, Instrument, Site, or 
+name of user who uploaded the file.  Partial string matching can be used on many 
```

Comment:
Under the Browse tab, use the Selection Filters to search for files by fields such as filename, file type, candidate PSCID, Visit Label, Instrument, Site, or name of user who uploaded the file.